windows/internal/copy.bat

```diff
@@ -10,3 +10,4 @@ copy "%CUDA_PATH%\bin\nvrtc*64_*.dll*" pytorch\torch\lib
 
 copy "C:\Program Files\NVIDIA Corporation\NvToolsExt\bin\x64\nvToolsExt64_1.dll*" pytorch\torch\lib
 copy "%CONDA_LIB_PATH%\libiomp*5md.dll" pytorch\torch\lib
+copy "%CONDA_LIB_PATH%\uv.dll" pytorch\torch\lib
```

Comment:
Does this mean that the windows distribution of Pytorch will actually still contain uv.dll inside of it?